code/api/README.md

```diff
@@ -31,8 +27,21 @@ value: 'Bearer ${token}' where token is the user's authorization token
         "id": 6,  
         "name": "test",  
         "email": "test@test.com",  
-        "style": "classy"  
+        "style": "Punk"  
       }  
```

Comment:
Great - December and I have updated on the FE to sort the results, and we will send you the top ranked style

Reply:
Right on, thanks for being flexible!